resources/views/setpassword.blade.php

```diff
@@ -0,0 +1,142 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="csrf-token" content="{{ csrf_token() }}" />
+    <title> {{__('lang.Set_Password')}} </title>
+
+    <!-- Styles -->
```

Comment:
create separate header and footer files, and write CSS in the header and Js in the footer. Then extends those file wherever u want it